src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,57 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
+	if (
+		!L.CheckParamNumber(2) ||
+		!L.CheckParamNumber(3) ||
+		!L.CheckParamNumber(4)
```

Comment:
Can be replaced with `CheckParamNumber(2, 4)` the functions accepts a range to check.